webrtc.html

```diff
@@ -5521,6 +5521,23 @@ <h3><dfn>RTCRtpSender</dfn> Interface</h3>
           <code><a data-link-for="RTCRtpSender">setParameters</a></code>, even
           when simulcast isn't used.</div>
         </li>
+        <li>
+          <p>Let <var>maxN</var> be the maximum number of total simultaneous
+          encodings the user agent supports for this <var>kind</var>, at minimum
+          <code>1</code>.</p>
```

Comment:
Problematic: The number of supported simulcast streams can be different between codecs.

Reply:
One possible solution: set `active` to false for any unsupported encodings. This could take effect when SRD is called, and would give the application the opportunity to attempt to re-enable them by setting `active` to true and changing `codecPayloadType`.